go.mod

```diff
@@ -11,9 +11,9 @@ require (
 )
 
 require (
+	github.com/google/go-cmp v0.5.7 // indirect
 	golang.org/x/sys v0.0.0-20220520151302-bc2c85ada10a // indirect
 	golang.org/x/text v0.3.7 // indirect
 	google.golang.org/genproto v0.0.0-20220525015930-6ca3db687a9d // indirect
 	google.golang.org/protobuf v1.28.0 // indirect
-	gopkg.in/yaml.v3 v3.0.0 // indirect
```

Comment:
this needs to stay because otherwise we have a security issue (unless other deps got updated that makes it unneeded now)

Reply:
Hmm, I did a `go mod tidy` and it got removed. So I sort of wonder 🤔